doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -1,13 +1,24 @@
 [id='test-designer-run-test-proc']
-= Running a test scenario
+= Running the test scenarios
 
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
-. The *Test Report* panel pops out from the right of the designer and displays the test result overview and the scenario status.
-. After the tests are executed, click *View Alerts* to open the *Alerts* panel at the bottom of the designer for the error messages.
-. In case the tests fails, review the error messages. Messages with a red cross indicate test failures where as those with a green check mark indicate successful tests.
+. To run the defined test scenarios, choose one of the following methods:
+* In the upper-right corner of your project page, click *Test* button to execute all the test scenarios available in your project inside multiple assets.
```

Comment:
s/click *Test* button/click *Test*
or if it is an icon
s/click *Test* button/click the *Test* icon

Reply:
Ack

Reply:
click *Test*. According to the style guide (page 212):

Refer to a button by its label only or tooltip name only; do not use button or push button.

This goes for all other instances as well.